project/plugins.sbt

```diff
@@ -5,40 +5,25 @@ resolvers += Classpaths.typesafeReleases
 // which is used by plugin "org.kohsuke" % "github-api" % "1.68"
 resolvers += "Bintray Jcenter" at "https://jcenter.bintray.com/"
 
-libraryDependencies += "org.kohsuke" % "github-api" % "1.68"
+// for sbt-bintray for resolving credentials from env vars
+resolvers += Resolver.url("2m-sbt-plugin-releases", url("https://dl.bintray.com/2m/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
-
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.9")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.0")
 addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.2.6")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.1")
 addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.3.3")
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.0")
-
+addSbtPlugin("com.dwijnand" % "sbt-dynver" % "1.0.0+15-3655d014")
```

Comment:
I'll make a proper release of this (hopefully this weekend), so I can guarantee compatibility (using MiMa).